### DISCUSSION TITLE
/create-a-compose-form-mail-add-in-for-outlook-with-the-office-developer-tools-update-2/

### DISCUSSION THREAD

**Discussion by comment-migrator** [estruyf/eliostruyf-comments#224] [Comments]
Create a compose form mail add-in for Outlook with the Office Developer Tools update 2
        
[https://www.eliostruyf.com/create-a-compose-form-mail-add-in-for-outlook-with-the-office-developer-tools-update-2/](https://www.eliostruyf.com/create-a-compose-form-mail-add-in-for-outlook-with-the-office-developer-tools-update-2/)

**Comment by alex-bucur** [comment]
Great article! You just saved me from many headahches, thank you!

**Reply by estruyf** [reply]
Great to hear that this is still relevant!
